lib/ansible_tower_client/job_template.rb

```diff
@@ -14,7 +14,8 @@ def initialize(raw_body)
     def launch(vars = {})
       launch_url = "#{url}launch/"
       extra = JSONValues.new(vars).extra_vars
-      resp = Api.post(launch_url, extra).body
+      extra_vars = {"extra_vars": extra.to_json}
```

Comment:
@bdunne - Yeah - Then I'll have to fix the issue in JSONValues - I believe it is currently doing a JSON.parse on extra_vars and returning as a native ruby hash - which isn't correct. It probably should see if the parse doesn't blow up - and then return what was sent.